sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnostics.java

```diff
@@ -113,7 +113,6 @@ public Set<URI> getRegionsContacted() {
      *
      * @return set of regions contacted for this request
      */
```

Comment:
Regarding the API -> 
`getRegionsContacted()` - it has been long deprecated, should we remove it?

Reply:
I don't want to remove anything in this release (walmart recommended - tight timeline) - let's remove deprecated APIs after holiday season - sometime in Q1